stake-o-matic.sh

```diff
@@ -83,6 +83,7 @@ MAINNET_BETA_ARGS=(
   --min-self-stake-exceptions-key ${SELF_STAKE_EXCEPTIONS_KEY:?}
   --performance-db-url ${PERFORMANCE_DB_URL:?}
   --performance-db-token ${PERFORMANCE_DB_TOKEN:?}
+  --performance-waiver-release-version 1.14.0
```

Comment:
We might as well make this 1.14.14 since previously releases are not compatible with mainnet.